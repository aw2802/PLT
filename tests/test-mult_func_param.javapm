public class TwoFunctionsWithParameters {
	public void main() {
		println("Hi");
		int printMe = say5(true);
		println(printMe);
	}
	
	public int say5(boolean x) {
		println(x);
		return 1;
	}
}
