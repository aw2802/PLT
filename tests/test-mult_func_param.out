Hi
5
5
